packages/@react-aria/dnd/docs/useDraggableCollection.mdx

```diff
@@ -0,0 +1,658 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-aria/dnd';
+import statelyDocs from 'docs:@react-stately/dnd';
+import sharedDocs from 'docs:@react-types/shared';
+import selectionDocs from 'docs:@react-aria/selection';
+import {HeaderInfo, FunctionAPI, TypeContext, InterfaceType, TypeLink, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-aria/dnd/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Drag and Drop
+keywords: [drag, drop, dnd, drag and drop, aria, accessibility]
+---
+
+# useDraggableCollection
```

Comment:
examples on this page aren't getting rendered?

Reply:
I think I meant to leave this on `useDroppableCollection`. Not seeing them there.